alvr/client_core/src/lib.rs

```diff
@@ -78,6 +78,11 @@ pub struct ClientCapabilities {
     pub encoder_high_profile: bool,
     pub encoder_10_bits: bool,
     pub encoder_av1: bool,
+    pub use_10bit: bool,
+    pub use_full_range: bool,
+    pub encoding_gamma: f32,
+    pub enable_hdr: bool,
+    pub enable_color_correction: bool,
```

Comment:
I don't understand adding `enable_color_correction` here. it's not even defining a specific color correction profile. I think it should be left out, as color correction should be treated as a "deviation" from an already color corrected image.

Reply:
Honestly my main complaint w/ color correction is that it's currently enabled by default with settings specific to some particular headset, when it probably shouldn't be default on at all

Reply:
Then let's disable it by default